File: visualBasic/Ice/bidir/CallbackReceiverI.vb
' **********************************************************************
'
' Copyright (c) 2003-2017 ZeroC, Inc. All rights reserved.
'
' **********************************************************************

Imports BidirDemo

Public NotInheritable Class CallbackReceiverI
    Inherits CallbackReceiverDisp_

    Public Overloads Overrides Sub callback(ByVal num As Integer, ByVal current As Ice.Current)
        System.Console.Out.WriteLine("received callback #" & num)
    End Sub

End Class

File: java/Glacier2/callback/CallbackReceiverI.java
// Copyright (c) ZeroC, Inc.

import com.zeroc.demos.Glacier2.callback.Demo.*;

public final class CallbackReceiverI implements CallbackReceiver
{
    @Override
    public void callback(com.zeroc.Ice.Current current)
    {
        System.out.println("received callback");
    }
}
